package.json

```diff
@@ -95,6 +93,10 @@
     "node": "4.x || 5.x || 6.x || 7.x",
     "npm": "2.x || 3.x || 4.x"
   },
+  "dependencies": {
```

Comment:
Why did you move these? I think dev should be fine. This package.json is only used for development; we have others for the npm packages that get released.